packages/openapi3-parser/lib/parser/oas/parseServerVariableObject.js

```diff
@@ -20,9 +20,15 @@ const parseEnumStrings = R.curry((context, element) => {
   const parseEnumString = R.unless(isString,
     createWarning(namespace, `'${name}' 'enum' array value is not a string`));
 
+  const validateArrayNotEmpty = R.when(
+    array => array.isEmpty,
```

Comment:
As this may be suggested in review, `R.isEmpty` won't work because it doesn't appear to work for ArrayElement.